module/os/linux/zfs/abd_os.c

```diff
@@ -1340,6 +1340,8 @@ abd_bio_map_off(struct bio *bio, abd_t *abd,
 	return (io_size);
 }
 
+EXPORT_SYMBOL(abd_alloc_from_pages);
```

Comment:
I am not very familiar with those Linux macros, but shouldn't we export `abd_free()` and others then to make it a reasonably complete KPI?  May be I am missing something, but I can't find any `abd*`symbols exported.

Reply:
Good catch. We will need `abd_free()` exported as well. I updated the PR.